lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -18,6 +18,9 @@ import (
 	"fmt"
 	"sort"
 
+	feg "magma/feg/cloud/go/feg"
```

Comment:
Importing from the feg to lte module is something we have purposely avoided as it creates a cyclic dependency between modules. Previously only feg could import from lte.